CHANGELOG.md

```diff
@@ -1,16 +0,0 @@
-# [1.1.0](https://github.com/dhis2/datastore-app/compare/v1.0.0...v1.1.0) (2021-02-12)
```

Comment:
This shouldn't be deleted, I don't think...

Reply:
Oooops :sweat_smile: